xray/client/package.json

```diff
@@ -5,16 +5,25 @@
   "main": "index.js",
   "scripts": {
     "build": "rollup -c",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write **/*.ts **/*.js"
   },
   "author": "Holger Rapp <hrapp@lyft.com>",
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/three": "0.89.0",
+    "prettier": "^1.10.2",
     "rollup": "^0.55.3",
     "rollup-plugin-commonjs": "^8.3.0",
     "rollup-plugin-node-resolve": "^3.0.2",
     "rollup-plugin-typescript": "^0.8.1",
     "three": "0.89.0"
+  },
+  "prettier": {
+    "arrow-parens": "always",
+    "bracket-spacing": false,
+    "print-width": 80,
+    "single-quote": true,
+    "trailing-comma": "es5"
```

Comment:
in typescript we use es6 for output. Should this be consistent?

Reply:
For JS code I think we want `"es5"` here — it's a bit weird but the only options are `"none"`, `"es5"`, and `"all"`: https://prettier.io/docs/en/options.html#trailing-commas. If it's set to `"all"`, it will insert trailing commas in function parameter lists. That's not valid ES6 (though it is supported in TypeScript, apparently).

I could enable it for the TypeScript if you want though.

Reply:
that's fine.